introduction.md

```diff
@@ -1,2 +1,5 @@
 Hello, my name is Joakim.
-My favorite Code School path is  HTML/CSS.
\ No newline at end of file
+My favorite Code School path is  HTML/CSS.
+* HTML
+* CSS
+* PHP
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?